test/700_status_and_report_test.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+
+. ./config.sh
+
+start_suite "weave status/weave report"
+
+weave_on $HOST1 launch --nickname nicknamington
+
+start_container $HOST1 --name test
+
+# Assert report template $2 has value $3 on host $1
+assert_report() {
+    assert "weave_on $1 report -f '$2'" "$3"
+}
+
+# Assert status property $2 has value $3 on host $1
+assert_status() {
+    assert "weave_on $1 status | grep -oP '(?<= $2: ).*'" "$3"
+}
+
+assert_report $HOST1 "{{.Router.NickName}}"                       "nicknamington"
+assert_report $HOST1 "{{len .Router.Peers}}"                      "1"
+assert_report $HOST1 "{{.IPAM.Range}}"                            "[10.32.0.0-10.48.0.0)"
+assert_report $HOST1 "{{.IPAM.DefaultSubnet}}"                    "10.32.0.0/12"
```

Comment:
I'd rather not hard-code the default range into the tests. Either find some other, less volatile, value to check, or start the router with an explicit range.